src/node_api_helpers.h

```diff
@@ -24,6 +24,13 @@
 #define NAPI_MODULE_INIT(name)                                                 \
   void name(napi_env env, napi_value exports, napi_value module)
 
+#define NAPI_GETTER(name)                                                      \
+  void name(napi_env e, napi_propertyname property, napi_callback_info info)
+
+#define NAPI_SETTER(name)                                                      \
+  void name(napi_env e, napi_propertyname property, napi_value value,          \
+            napi_callback_info info)
```

Comment:
I'd like to discuss your motivations for this change. These macros are already defined just above, to be the same as `NAPI_METHOD`. Giving them signatures that don't match `napi_callback` is going to cause conflicts with APIs that use `napi_property_descriptor`. When I added the accessors implementation, I considered having accessor callbacks follow this pattern (to more closely follow Nan), but it made the NAPI api more complicated while not actually providing any added functionality.